docs/docs/reference/data_preparation/data_filtering.md

```diff
@@ -41,9 +42,46 @@ This macro allows you remove duplicate rows from a dbt model based on certain co
  2   |  completed   |    13:05:23     |
  3   |  completed   |    13:30:00     |
 ```
+### [filter_get_duplicates](https://re-data.github.io/dbt-re-data/#!/macro/macro.re_data.filter_get_duplicates)
+*Arguments:*
+- *relation: dbt base model to perform the filtering on*
+- *unique_cols: List of columns that uniquely identify each row*
+- *sort_columns: Order in which we want to sort the partitioned rows. e.g. (created_at DESC, created_at ASC to choose the latest or earliest row based on the timestamp column*
+
+Return type: table with duplicate rows
+
+along with the fields of the base model the macro returns duplication context in new fields:
+re_data_duplicates_count - total number of duplicates with the same current key set
+re_data_duplicate_row_number - number of current duplicate row inside the group of duplicates with the same current key set
+
 
+This macro allows you to get duplicate rows from a dbt model based on certain conditions.
+
+```sql
+  id |  status      |   updated_at    |
+--------------------------------------+
+ 1   |  pending     |    13:00:45     |
+ 2   |  completed   |    13:05:23     |
+ 1   |  completed   |    13:10:35     |
+ 2   |  pending     |    13:04:49     |
+ 3   |  completed   |    13:30:00     |
+
+ => select id, status, updated_at,
+	   re_data_duplicates_group_rows_count, 
```

Comment:
btw for rows_count, I noticed re_data in other places uses `row_count` so maybe we change in here (in docs) and add small PR regarding this for recently merged changes in dbt-re-data repo? @sergey-vdovin what do you think?

Reply:
[as i remember we discussed that while in the rest of the word](https://re-data.slack.com/archives/D02J4S82BU2/p1636642008002400

Sergey: Hi ) probably we should use row_count not rows_count like in the rest of dbt and outside of dbt, the same applies to duplicates_count
https://ell.stackexchange.com/questions/162678/row-count-or-rows-count-to-mean-number-of-rows
What do you think about it?)

Mateusz: yea probably, only thing worrying in changing this is what to do with existing metrics which are already generated)